docs/filesystem.md

```diff
@@ -1,4 +1,4 @@
-A 9p filesystem is supported by the emulator, using a virtio transport. Using
+A 9p file system is supported by v86, using a virtio transport. Using
```

Comment:
"filesystem" is correct (see https://en.wikipedia.org/wiki/File_system), and more consistent with the rest of the docs.